files/image_config/updategraph/updategraph

```diff
@@ -60,13 +60,20 @@ if [ -f /tmp/pending_config_migration ]; then
     copy_config_files_and_directories $copy_list
     if [ x"${WARM_BOOT}" == x"true" ]; then
         echo "Warm reboot detected..."
-    elif [ "$enabled" = "true" ]; then
+    elif [ -r /etc/sonic/config_db.json ]; then
+        echo "Use config_db.json from old system..."
+        sonic-cfggen -j /etc/sonic/config_db.json --write-to-db
+
+        if [[ -x /usr/bin/db_migrator.py ]]; then
```

Comment:
Just noting the use of double square brackets here but single brackets elsewhere. I think we need to decide on one to use moving forward.

Reply:
Yes. this file has fixed use of [] and [[ ]]. We need to unify the usage. I think [[ ]] is safer for some reason I don't remember now. But it could be just a myth :-)